.changelog/df93fff8f662441fa12dac87614a5064.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "df93fff8-f662-441f-a12d-ac87614a5064",
+    "type": "bugfix",
+    "description": "Add a setter to output checksum middleware cfg to enable request validation mode by default",
```

Comment:
Is this setter something that customers can interact with? If not, I'd just leave it as "Enable request validation mode by default"